### PR TITLE
chore: add node 18 support into the V2 template 

### PR DESCRIPTION
**Pull request by XiaofuHuang**
- Update the supported Node.js version in the documents
  - README of all the template and samples
  - Update 'WEBSITE_NODE_DEFAULT_VERSION' in the bicep file

Detail: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/16936740/

### CODE REVIEW

**Review by kuojianlu — APPROVED** [review]

File: templates/function-base/js/default/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Azure Functions are a great way to add server-side behaviors to any Teams applic
 
 ## Prerequisites
 
-- [NodeJS](https://nodejs.org/en/)
+- [Node.js](https://nodejs.org/en/), supported versions: 14, 16, 18
```
Comment (hund030) [inline templates/function-base/js/default/README.md:9]: Is this also preview?

File: packages/fx-core/templates/plugins/resource/spfx/solution/README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ The SharePoint Framework (SPFx) is a page and web part model that provides full
 
 ## Prerequisites
 
-> - Node.js v16
+- [Node.js](https://nodejs.org/en/), supported versions: 16
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/templates/plugins/resource/spfx/solution/README.md:18]: Please also help update README in SPFx template under `src` folder.

File: packages/fx-core/templates/bicep/azureWebApp.provision.module.bicep [hunk 3]
```diff
@@ -39,7 +39,7 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
         {{#if (contains "node" configs)}}
         {
           name: 'WEBSITE_NODE_DEFAULT_VERSION'
-          value: '~16' // Set NodeJS version to 16.x for your site
+          value: '~18' // Set NodeJS version to 16.x for your site
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/templates/bicep/azureWebApp.provision.module.bicep:42]: Also update comment?

**Review by swatDong — DISMISSED** [review]

File: templates/function-base/js/default/README.md (hunk 1 above)
Comment (XiaofuHuang) [reply templates/function-base/js/default/README.md]: Updated, Thx.

File: packages/fx-core/templates/plugins/resource/spfx/solution/README.md (hunk 2 above)
Comment (XiaofuHuang) [reply packages/fx-core/templates/plugins/resource/spfx/solution/README.md]: Updated, thx.

File: packages/fx-core/templates/bicep/azureWebApp.provision.module.bicep (hunk 3 above)
Comment (XiaofuHuang) [reply packages/fx-core/templates/bicep/azureWebApp.provision.module.bicep]: Updated, thx.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: packages/fx-core/templates/plugins/resource/spfx/solution/README.md (hunk 2 above)
Comment (XiaofuHuang) [reply packages/fx-core/templates/plugins/resource/spfx/solution/README.md]: Maybe it is for V3. Here we only change v2 templates

**Review by swatDong — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Review by Siglud — APPROVED** [review]

**Pull request merged**